frontend/src/main/components/Commons/Profits.js

```diff
@@ -1,24 +1,34 @@
 import React from "react";
 import { Card } from "react-bootstrap";
 import ProfitsTable from "main/components/Commons/ProfitsTable"
+import moment from "moment";
 
+function timestampToDate(timestamp) {
+    return moment(timestamp).format('YYYY-MM-DD');
+}
+
+/*
 const dummyData = [
     { id: 1, profit: 10, date: "2021-03-05" },
     { id: 2, profit: 11, date: "2021-03-06" },
     { id: 3, profit: 10, date: "2021-03-07" },
     { id: 4, profit: 8, date: "2021-03-08" }
-];
+]; */
```

Comment:
Do we need this commented-out code? I believe we might be able to get rid of this since we changed the profits table param from "DummyData" to "profitsForTable"

Reply:
Thanks for pointing that out. I have deleted the commented-out code.